gen/input-output-hk/cardano-node/v8.0.0/detail/configuration.yaml

```diff
@@ -0,0 +1,265 @@
+- title: 'workbench:  updates'
+  author: Kosyrev Serge
+  url: https://github.com/input-output-hk/cardano-node/pull/5065
+  number: 5065
+  mergedAt: "2023-04-12T00:59:51Z"
+  files:
+    - .gitignore
+    - Makefile
+    - bench/locli/locli.cabal
+    - bench/locli/src/Cardano/Analysis/API.hs
+    - bench/locli/src/Cardano/Analysis/API/Types.hs
+    - bench/locli/src/Cardano/Analysis/BlockProp.hs
+    - bench/locli/src/Cardano/Analysis/Summary.hs
+    - bench/locli/src/Cardano/Command.hs
+    - bench/locli/src/Cardano/Render.hs
+    - bench/locli/src/Cardano/Unlog/LogObject.hs
+    - bench/locli/src/Cardano/Util.hs
+    - bench/locli/src/Data/CDF.hs
+    - bench/locli/src/Data/Profile.hs
+    - bench/workbench.gnuplot
+    - configuration/cardano/membench-config-new.yaml
+    - lib.mk
+    - nix/haskell.nix
+    - nix/nixos/cardano-node-service.nix
+    - nix/pkgs.nix
+    - nix/workbench/analyse/analyse.sh
+    - nix/workbench/backend/runner.nix
+    - nix/workbench/ede/chart.ede
+    - nix/workbench/lib-cabal.sh
+    - nix/workbench/profile/prof1-variants.jq
+    - nix/workbench/profile/profile-run.jq
+    - nix/workbench/profile/profile.nix
+    - nix/workbench/service/generator.nix
+    - nix/workbench/service/nodes.nix
+    - nix/workbench/service/tracing.nix
+    - nix/workbench/shell.nix
+    - nix/workbench/wb
+  include: undecided
```

Comment:
Most of the PRs marked `include: undecided` are things I haven't decided whether to include in the changelog because they from outside the node team.